Build TestPlatform.vsix with RC3 VSSDK tooling

Faizan2304 opened the pull request.

Commit:
Localize nuget packages.

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge remote-tracking branch 'upstream/master'

Faizan2304 committed:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Merge remote-tracking branch 'upstream/master'

Commit:
Build vsix using RC3 VSSDK tooling.

msftclas commented:
Hi __@Faizan2304__, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Faizan Ahmad). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla.microsoft.com.
    </p>

TTYL, MSBOT;

Faizan2304 committed:
Merge branch 'master' into vsixBuildwithRC3

harshjain2 committed:
Merge branch 'master' into vsixBuildwithRC3

Commit:
Issue: https://devdiv.visualstudio.com/DefaultCollection/DevDiv/_workitems?id=366417&_a=edit

Fix: Update manifest.json to have entry for all the files and theit sha256 which are going to be part of vsix.

Testing: Manually validated by installing new vsix. The old folder is getting deleted after installing new one.

Commit:
Merge branch 'vsixBuildwithRC3' of https://github.com/faizan2304/vstest into vsixBuildwithRC3

Faizan2304 committed:
Merge branch 'master' into vsixBuildwithRC3

Commit:
Fix vsix folder name in extension folder with name "TestPlatform"

Commit:
nitpick: Spelling mistake

Faizan2304 committed:
Merge branch 'master' into vsixBuildwithRC3
